pkgs/development/tools/backblaze-b2/default.nix

```diff
@@ -1,17 +1,18 @@
 { fetchFromGitHub, makeWrapper, pythonPackages, stdenv }:
 
 pythonPackages.buildPythonApplication rec {
-  name = "backblaze-b2-${version}";
-  version = "0.6.2";
+  pname = "b2";
```

Comment:
I don't see the value of separating this out.

Reply:
Fair point, this came from following templates within python-modules.nix before I found backblaze-b2 as it's own package definition in tools. I'll remove the unnecessary breakout.